---
title: System.ServiceModel.AsyncCallbackThrewException
ms.date: 03/30/2017
ms.assetid: dfc27ca7-b904-4f29-a5b6-dc702fa54a20
ms.openlocfilehash: 3b9f1375f2d302bcf182743e94dd595a389d88ef
ms.sourcegitcommit: bc293b14af795e0e999e3304dd40c0222cf2ffe4
ms.translationtype: MT
ms.contentlocale: es-ES
ms.lasthandoff: 11/26/2020
ms.locfileid: "96262988"
---
# <a name="systemservicemodelasynccallbackthrewexception"></a><span data-ttu-id="eb8b2-102">System.ServiceModel.AsyncCallbackThrewException</span><span class="sxs-lookup"><span data-stu-id="eb8b2-102">System.ServiceModel.AsyncCallbackThrewException</span></span>

<span data-ttu-id="eb8b2-103">System.ServiceModel.AsyncCallbackThrewException</span><span class="sxs-lookup"><span data-stu-id="eb8b2-103">System.ServiceModel.AsyncCallbackThrewException</span></span>  
  
## <a name="description"></a><span data-ttu-id="eb8b2-104">Descripción</span><span class="sxs-lookup"><span data-stu-id="eb8b2-104">Description</span></span>  

 <span data-ttu-id="eb8b2-105">Una devolución de llamada asincrónica produjo una excepción.</span><span class="sxs-lookup"><span data-stu-id="eb8b2-105">An asynchronous callback threw an exception.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="eb8b2-106">Vea también</span><span class="sxs-lookup"><span data-stu-id="eb8b2-106">See also</span></span>

- [<span data-ttu-id="eb8b2-107">Seguimiento</span><span class="sxs-lookup"><span data-stu-id="eb8b2-107">Tracing</span></span>](index.md)
- [<span data-ttu-id="eb8b2-108">Uso del seguimiento para solucionar problemas de su aplicación</span><span class="sxs-lookup"><span data-stu-id="eb8b2-108">Using Tracing to Troubleshoot Your Application</span></span>](using-tracing-to-troubleshoot-your-application.md)
- [<span data-ttu-id="eb8b2-109">Administración y diagnóstico</span><span class="sxs-lookup"><span data-stu-id="eb8b2-109">Administration and Diagnostics</span></span>](../index.md)
